R/fit.R

```diff
@@ -40,6 +40,16 @@ fit_single_optimizer <- function(formula,
                                  formula_parts,
                                  ...,
                                  control = mmrm_control(...)) {
+  to_remove <- list(
+    # Transient visit to invalid parameters.
+    warnings = c("NA/NaN function evaluation")
+  )
+  as_diverged <- list(
+    errors = c(
+      "NA/NaN Hessian evaluation",
+      "L-BFGS-B needs finite values of 'fn'"
+    )
+  )
```

Comment:
if this is something static, should it be defined somewhere not inside the function but in the package environment?

Reply:
I think it is ok, we had it like that before too for the warnings to remove - it is not used by any other functions e.g.